bindings/python/edlib.pyx

```diff
@@ -3,10 +3,11 @@ from cpython.mem cimport PyMem_Malloc, PyMem_Free
 
 cimport cedlib
 
+
 def align(query, target, mode="NW", task="distance", k=-1, additionalEqualities=None):
     """ Align query with target using edit distance.
-    @param {string or bytes array} query
-    @param {string or bytes array} target
+    @param {str or bytes or iterable of hashable objects} query
+    @param {str or bytes or iterable of hashable objects} target
```

Comment:
Let's mention here the limit of 255 different unique values.

Reply:
Actually, it's 256, I had an off-by-one error in the previous commit :-)